packages/shared/prisma/schema.prisma

```diff
@@ -570,3 +576,77 @@ model AuditLog {
     @@index([createdAt])
     @@map("audit_logs")
 }
+
+model EvalTemplate {
+    id        String   @id @default(cuid())
+    createdAt DateTime @default(now()) @map("created_at")
+    updatedAt DateTime @default(now()) @updatedAt @map("updated_at")
+
+    projectId String?  @map("project_id") // null for global templates managed by langfuse
```

Comment:
interesting that they are save din db, first intuition would be to store templates in application code (learning from how painful it is to update this when migrations are needed, see pricings)

any advantages of having this here? afaik templates are not part of this initial pr anyway